base/static/base/css/_variables.scss

```diff
@@ -87,7 +87,7 @@ $trans-orange: rgba(154, 83, 46, 0.8);
 // Table Color Scheme
 
 $table-main: #ADB17D;
-$table-accent: ;
+//$table-accent: ;
```

Comment:
I just commented it out because it was @kzadrozny's SASS and I thought it might serve as a reminder for something she was working on prior but might not have finished. It could just as well have been deleted. It does have to go away though because it causes the new version of the libsass compiler to fail.